spec/support/queues_helper.rb

```diff
@@ -8,4 +8,17 @@ def webhook_queue
       :webhook
     end
   end
+
+  # This performs any enqueued-jobs, and continues doing so until the queue is empty.
+  # Lots of the jobs enqueue other jobs as part of their work, and this ensures that
+  # everything that's supposed to happen, happens.
+  #
+  # ⚠️ Notice that `have_been_enqueued` might not work with perform_all_enqueued_jobs
+  # because it's only aware of the last run of the loop.
+  def perform_all_enqueued_jobs
```

Comment:
I moved `perform_all_enqueued_jobs` outside `ScenariosHelper` since I'm using it outside the scenario folder.